lib/taskana-core/src/main/java/pro/taskana/task/api/TaskQuery.java

```diff
@@ -536,6 +536,15 @@ public interface TaskQuery extends BaseQuery<TaskSummary, TaskQueryColumnName> {
    */
   TaskQuery orderByState(SortDirection sortDirection);
 
+  /**
+   * Add the hasComments flag to the query.
+   *
+   * @param hasComments as Boolean. If null, it won't be integrated into the statement. You have to
+   *     set false.
```

Comment:
Please remove the dot after "false"

Reply:
okay. There is also a dot after "false" in the documentation of the method readEquals. Should i delete it there too?